pycodestyle.py

```diff
@@ -1150,6 +1150,37 @@ def comparison_type(logical_line, noqa):
         yield match.start(), "E721 do not compare types, use 'isinstance()'"
 
 
+def ambiguous_identifier(logical_line, tokens):
+    r"""Never use the characters 'l', 'O', or 'I' as variable names.
+
+    In some fonts, these characters are indistinguishable from the numerals
+    one and zero. When tempted to use 'l', use 'L' instead.
+
+    Okay: L = 0
+    E741: l = 0
```

Comment:
Thanks @mtmiller ! Could you update the doc tests with the rest of the cases that are not allowed?

Also, the 'as' test cases are missing.